apps/condo/domains/notification/transports/push.js

```diff
@@ -47,10 +57,19 @@ async function getTokens (ownerId, id) {
     const appIds = {}
 
     if (!isEmpty(remoteClients)) {
-        remoteClients.forEach(({ pushToken, pushType, pushTransport, appId }) => {
-            tokensByTransport[pushTransport].push(pushToken)
-            pushTypes[pushToken] = pushType
-            appIds[pushToken] = appId
+        remoteClients.forEach((remoteClient) => {
+            const {
+                appId,
+                pushToken, pushType, pushTransport,
+                pushTokenVoIP, pushTypeVoIP, pushTransportVoIP,
+            } = remoteClient
+            const transport = isVoIP ? pushTransportVoIP : pushTransport
+            const token = isVoIP ? pushTokenVoIP : pushToken
+            const type = isVoIP ? pushTypeVoIP : pushType
```

Comment:
Dynamic object destructing :)
```suggestion
            const {
                appId,
                [isVoIP ? pushTransportVoIP : pushTransport]: transport,
                [isVoIP ? pushTokenVoIP : pushToken]: token,
                [isVoIP ? pushTypeVoIP : pushType]: type,
            } = remoteClient
```

Reply:
linter error here :(